src/qmapshack/helpers/CMapIconSizesSetup.cpp

```diff
@@ -0,0 +1,83 @@
+/**********************************************************************************************
+    Copyright (C) 2014 Oliver Eichler <oliver.eichler@gmx.de>
```

Comment:
Just noticed: Not my work & copyright :wink:  If you enter your name you will get part of the credits (which is well deserved by now)